css/css-transforms/skewX/svg-skewx-020.html

```diff
@@ -22,7 +22,7 @@
     <p>The test passes if there is a green square and no red.</p>
     <svg>
         <path d="M 2,1 100,1 198,99 101,99 Z" fill="red"/>
-        <rect width="100" height="100" fill="green" transform="skewX(0.0125e1turn)"/>
+        <rect width="100" height="100" fill="green"  style="transform: skewX(0.0125e1turn)"/>
```

Comment:
Looks like you have two spaces in lots of places.

Reply:
fixed